packages/devtools_shared/lib/src/deeplink/app_link_settings.dart

```diff
@@ -9,16 +9,22 @@ import 'dart:convert';
 
```

Comment:
please add updates to the devtools_shared changelog for the changes to this package